modules/notification/base/notifier.go

```diff
@@ -22,6 +23,7 @@ type Notifier interface {
 	NotifyTransferRepository(doer *user_model.User, repo *repo_model.Repository, oldOwnerName string)
 	NotifyNewIssue(issue *models.Issue, mentions []*user_model.User)
 	NotifyIssueChangeStatus(*user_model.User, *models.Issue, *models.Comment, bool)
+	NotifyDeleteIssue(*user_model.User, *git.Repository, *models.Issue)
```

Comment:
```suggestion
	NotifyDeleteIssue(*user_model.User, *models.Issue)
```
Is there a reason we need to pass the git repo down to all notifyers ?!?

Reply:
Hmm, this was added by lunny and was not included in my original commented-out stub. In the end, this just does nothing as of now, so I think it does not really matter if we change or keep it. Whoever implements the real notifier can then change or keep it here, as required.

The NotifyDeleteComment also works without passing a Git repo, and as far as i can see also doesn't load more data from the repo ID. Thus I think we can drop the git.Repository here and just pass the Issue.

Reply:
agree - I'll ajust things 

if really needed we can add it back at any time